beetsplug/lyrics.py

```diff
@@ -659,22 +663,69 @@ def commands(self):
             help=u'print lyrics to console',
         )
         cmd.parser.add_option(
+            u'-r', u'--write-rst', dest='writerst',
+            action='store', default='.',
+            help=u'write lyrics to given directory as RST files',
+        )
+        cmd.parser.add_option(
             u'-f', u'--force', dest='force_refetch',
             action='store_true', default=False,
             help=u'always re-download lyrics',
         )
+        cmd.parser.add_option(
+            u'-s', u'--skip', dest='skip_fetched',
+            action='store_true', default=False,
+            help=u'skip already fetched lyrics',
+        )
```

Comment:
that's true, the description is completely backwards. i was trying to make the opposite option of `--force`, so `--skip` made sense.  not sure what would be a better name... `-l` and `--local` seem better, but don't fit well with `--force`...